pkg/openshift/template.go

```diff
@@ -47,13 +47,22 @@ func ProcessTemplate(templateDir string, name string, paramDir string, compareOp
 		if err != nil {
 			return []byte{}, err
 		}
-		tempParamFile := ".combined.env"
-		defer os.Remove(tempParamFile)
-		cli.DebugMsg("Writing contents of param files into", tempParamFile)
-		err = ioutil.WriteFile(tempParamFile, paramFileBytes, 0644)
+		
+		tempParamFile, err := ioutil.TempFile("", ".combined.*.env")
 		if err != nil {
+			log.Fatal(err)
```

Comment:
`log` is not imported - better to return the error anyway I think: `return []byte{}, err`